### PR TITLE
escape _ symbol for markdown format

### PR DESCRIPTION
**Pull request by koluchiy**
Escape underscore symbol for message format markdown

For example, you have _ in your username and use default telegram message
Error: Bad Request: can't parse entities in message text: Can't find end of the entity starting at byte offset

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 .PHONY: install build test html update docker_build docker_image docker_deploy clean version
 
 VERSION := $(shell git describe --tags --always || git rev-parse --short HEAD)
-DEPLOY_ACCOUNT := "appleboy"
+DEPLOY_ACCOUNT := "koluchiy"
```
Comment (appleboy) [inline Makefile]: Don't change this.

File: plugin.go [hunk 2]
```diff
@@ -213,9 +234,25 @@ func (p Plugin) Exec() error {
 	locations := trimElement(p.Config.Location)
 	venues := trimElement(p.Config.Venue)
 
+	message = trimElement(message)
+
+	if p.Config.Format == "markdown" {
+		message = escapeMarkdown(message)
+
+		p.Build.Message = escapeMarkdownOne(p.Build.Author)
```
Comment (appleboy) [inline plugin.go]: Author?

Comment (appleboy) [inline plugin.go]: Why all filed need to use `escapeMarkdownOne` converter.

**Comment by appleboy** [comment]
build fail

File: Makefile (hunk 1 above)
Comment (koluchiy) [inline Makefile]: sorry, fixed

File: plugin.go (hunk 2 above)
Comment (koluchiy) [inline plugin.go]: all fields can has underscore and can used for message

**Comment by koluchiy** [comment]
build fails because of unexisting environment. How can I fix it? I think it is global project problem

**Comment by appleboy** [comment]
@koluchiy I will fix the build error tomorrow. Thanks for your contribution.

**Comment by appleboy** [comment]
@koluchiy Can you help to rebase on latest `master` branch?

**Comment by koluchiy** [comment]
Yes, of course. I will rebase and squash my commits

**Comment by appleboy** [comment]
please fix the lint error. http://drone.wu-boy.com/appleboy/drone-telegram/6

**Comment by koluchiy** [comment]
@appleboy, build success )

**Pull request merged**